app/controllers/cart_items_controller.rb

```diff
@@ -1,10 +1,24 @@
 class CartItemsController < ApplicationController
 
+    def index
+        @cart = current_user.cart.cart_items
```

Comment:
Remove instance variable

Also it should be named cart_items as we will get the cart_items for particular cart